container-guide/concepts/containers-bci-labels.xml

```diff
@@ -217,9 +215,8 @@
       The skopeo tool makes it possible to examine labels of an image without
       pulling it first. For example:
     </para>
-<screen>skopeo inspect docker://registry.suse.com/bci/bci-base:15.4 | \
-    jq '.Labels'
-    skopeo inspect docker://registry.suse.com/bci/bci-base:15.4 | \
-    jq '.Labels["com.suse.sle.base.supportlevel"]'</screen>
+<screen>skopeo inspect -f {{.Labels | json}} docker://registry.suse.com/bci/bci-base:15.5
```

Comment:
JFYI: As long as the output is for programmatic consumption, this is fine but for human consumption, the output, even while dumping JSON, is not pretty.